packages/editor-ui/src/components/Node.vue

```diff
@@ -426,6 +428,18 @@ export default mixins(
 			});
 		},
 
+		onClick(event: MouseEvent) {
+			this.callDebounced('onClickDebounced', { debounceTime: 300, trailing: true }, event);
+		},
+
+		onClickDebounced(event: MouseEvent) {
+			if (event.detail === 2) {
```

Comment:
It's not obvious(at least to me) that `event.detail === 2` is double click, perhaps a comment or naming this condition as a variable would be more clear
```
const isDoubleClick = event.detail === 2
if (isDoubleClick) {
....
}
```
Also, we might want to consider changing it to `event.detail > 2`, in cases where people click frantically :D and manage to click three or more times. IMO that should still count as double click